src/AudioToolbox/AudioFile.cs

```diff
@@ -213,6 +216,44 @@ public struct AudioFileMarker
 		}
 	}
 
+	[Introduced (PlatformName.UIKitForMac, 13,0)]
+	[NoWatch, iOS (13,0), Mac (10,15, onlyOn64 : true), TV (13,0)]
+	[StructLayout (LayoutKind.Sequential)]
+	public struct AudioPacketRangeByteCountTranslation
+	{
+		public long Packet;
+		public long PacketCount;
+		public long ByteCountUpperBound;
+	}
+
+	[Introduced (PlatformName.UIKitForMac, 13,0)]
+	[NoWatch, iOS (13,0), Mac (10,15, onlyOn64 : true), TV (13,0)]
+	[StructLayout (LayoutKind.Sequential)]
+	public struct AudioPacketRollDistanceTranslation
+	{
+		public long Packet;
+		public long RollDistance;
+	}
+
+	[Introduced (PlatformName.UIKitForMac, 13,0)]
+	[NoWatch, iOS (13,0), Mac (10,15, onlyOn64 : true), TV (13,0)]
+	[StructLayout (LayoutKind.Sequential)]
+	public struct AudioIndependentPacketTranslation
+	{
+		public long Packet;
+		public long IndependentlyDecodablePacket;
+	}
+
+	[Introduced (PlatformName.UIKitForMac, 13,0)]
+	[NoWatch, iOS (13,0), Mac (10,15, onlyOn64 : true), TV (13,0)]
+	[StructLayout (LayoutKind.Sequential)]
+	public struct AudioPacketDependencyInfoTranslation
+	{
+		public long Packet;
+		public uint IsIndependentlyDecodable;
```

Comment:
it's a 4 bytes bool `1 means that the specified packet is independently decodable; 0 means it's not`
you can `[Marshal]` it to be nicer :)

Reply:
Nice catch, will be fixed.